### PR TITLE
changing the default driver to Chrome driver

### PR DESCRIPTION
**Pull request by vjpujar**
Please merge the changes

### CODE REVIEW

**Comment by joefitter** [comment]
this LGTM - @JoeChapman @gxxm @bendiggle could one of you have a 👀 ?

File: acceptance_tests/features/support/env.rb [hunk 1]
```diff
@@ -21,18 +21,18 @@
...
-  Capybara.register_driver :poltergeist do |app|
-    Capybara::Poltergeist::Driver.new(app, :phantomjs => Phantomjs.path)
-  end
-  Capybara.default_driver = :poltergeist
+  Capybara.run_server = false
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app, :phantomjs => Phantomjs.path)
+  end
+  Capybara.default_driver = :poltergeist
 end
```
Comment (joefitter) [inline acceptance_tests/features/support/env.rb]: Why are these showing as changes? These lines are identical - could it be whitespace or line feeds or something? @JoeChapman ?

File: acceptance_tests/features/support/env.rb [hunk 2]
```diff
@@ -3,8 +3,8 @@
 require 'capybara/dsl'
 require 'rspec'
 require 'show_me_the_cookies'
-require 'capybara/poltergeist'
-require 'phantomjs'
+require 'capybara/poltergeist'
+require 'phantomjs'
```
Comment (joefitter) [inline acceptance_tests/features/support/env.rb]: and here

File: acceptance_tests/features/support/env.rb (hunk 1 above)
Comment (bendiggle) [inline acceptance_tests/features/support/env.rb]: @joefitter I'm not sure why but I'm having the same issue with a PR for the phantomJS version gemfile change on RTM...

**Comment by JoeChapman** [comment]
hmmm?

**Comment by JoeChapman** [comment]
@vjpujar what exactly does this do? Please describe what this PR is attempting to solve.

**Comment by easternbloc** [comment]
what @JoeChapman said. It'd be nice to see why this change is being made. Can you add a description explaining why you need to use chrome driver?

**Comment by easternbloc** [comment]
It helps for historical purposes.

**Comment by easternbloc** [comment]
And ideally that reasoning should be written into the commit itself.

**Comment by JoeChapman** [comment]
@vjpujar!

**Comment by lennym** [comment]
This is a valuable read on how to write useful git commit messages - http://www.philandstuff.com/2014/02/09/git-pickaxe.html

**Pull request closed without merging**